agasc/supplement/utils.py

```diff
@@ -74,6 +75,8 @@ def get_supplement_table(name, agasc_dir=None, as_dict=False):
     """
     agasc_dir = default_agasc_dir() if agasc_dir is None else Path(agasc_dir)
 
+    dtypes = defaultdict(lambda: [], {'bad': BAD_DTYPE, 'mags': MAGS_DTYPE, 'obs': OBS_DTYPE})
```

Comment:
I think this is typically written as `defaultdict(list, ..)`, no?  But more to the point is the even more common thing below to supply the default in a `.get`, as in below where the default is specified in the same line of code.
```
dat = np.array([], dtype=dtypes.get(name, []))
```